packages/components/src/sortTabs/SortTab.js

```diff
@@ -0,0 +1,174 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  Hoverable,
+  designTokens,
+} from '@kiwicom/universal-components';
+
+import TouchableWithoutFeedback from '../TouchableWithoutFeedback';
+import type { SortIconTypes, SortTypes, PriceDurationInfo } from './SortTabs';
+
+export type Props = {|
+  +label: string,
+  +value: SortTypes,
+  +isFirst: boolean,
+  +isLast: boolean,
+  +isActive: boolean,
+  +priceDurationInfo: ?PriceDurationInfo,
```

Comment:
What about `additionalInfo` instead of `priceDurationInfo`? 
Just Idea

Reply:
Done!